napari_hub_cli/checklist/analysis.py

```diff
@@ -291,3 +294,7 @@ def write_csv(rows, output_filename):
         writer = csv.DictWriter(csvfile, fieldnames=headers)
         writer.writeheader()
         writer.writerows(rows)
+    console = Console()
+    current_path = os.getcwd()
+    csv_path = current_path + '/' + output_filename
```

Comment:
Here, it's better to use `Path` from `pathlib` for this kind of things. Unfortunately, the code you wrote is not portable as it considers the `/` that is specific of UNIX-like systems. To avoid that, the `Path` class from `pathlib` is very useful :). 
You can first create an instance from the `output_filename`:

```python
from pathlib import Path

output_path = Path(output_filename)
```
And that's it :D you can directly call a `str` on it if you want the full path:

```python
csv_path = str(output_path)
```

or, you can directly use it in the f-string:

```python
f"\n CSV successfully created at {csv_path}"
```
By default in the f-string, python will use the `str(...)` on the expression that is in `{}`. 

Still using `Path` object, if you want to add a sub-path, you can do `output_path / 'mysubpath'` which automatically will use the right file separator depending on the system running your code.